.github/workflows/CI.yml

```diff
@@ -149,3 +149,18 @@ jobs:
           toolchain: stable
           override: true
       - run: cargo build --features=c-interface
+
+  test_freebsd:
+    runs-on: macos-latest
+    name: Check stable / FreeBSD host
+    steps:
+      - uses: actions/checkout@v2
+      - uses: vmactions/freebsd-vm@v0.1.4
+        id: test
+        with:
+          usesh: true
+          prepare: pkg install -y curl
+          run: |
+            curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --default-toolchain none -y
```

Comment:
[This](https://github.com/GuillaumeGomez/sysinfo/pull/478/files#diff-3ab46ee209a127470fce3c2cf106b1a1dbadbb929a4b5b13656a4bc4ce19c0b8R31-R36) cannot be used in here?

Reply:
What do you mean ? Tests ?

Reply:
No I mean for the installation of rust itself.

Reply:
Well anyway, this is fine as is, I'll maybe add some other checks later on.